hcatalog/core/src/main/java/org/apache/hive/hcatalog/mapreduce/FileOutputCommitterContainer.java

```diff
@@ -558,50 +557,67 @@ private void moveTaskOutputs(final Configuration conf, Path srcf, Path srcDir,
       }
     }
 
-    if (moves.isEmpty()) {
+    bulkMoveFiles(conf, srcFs, destFs, moves);
+  }
+
+  /**
+   * Bulk move files from source to destination.
+   * @param srcFs the source filesystem where the source files are
+   * @param destFs the destionation filesystem where the destionation files are
+   * @param pairs list of pairs of <source_path, destination_path>, move source_path to destination_path
+   * @throws java.io.IOException
+   */
+  private void bulkMoveFiles(final Configuration conf, final FileSystem srcFs, final FileSystem destFs, List<Pair<Path, Path>> pairs) throws IOException{
```

Comment:
I guess this line is quite long, isn't this reformatted by the style definition?

Reply:
thank you @abstractdog for pointing out the code violation. I was not using intelligi/eclipse that makes it harder to check the styles, so I was eyeballing. I downloaded intelligi and applied the eclipse-style.xml but there was some problem with the style xml (looks like some outdated format). The code is quite old and looks from pre-style check age so there are a lot warnings so I only fixed the part I changed. (also I was not using intelligi/eclipse that makes it harder to check the styles)
the new sonarcloud issues is from code not touched by this PR and test failures seem unrelated. can you help review again? thanks again!